# ampeers-fullstack-interview
## Exercice 5
### the log with be Error: BOOOM, with some error because the promise with be catched
